diff --git a/packages/core/src/schema.rs b/packages/core/src/schema.rs
index ee96e5c..4e1c9e4 100644
--- a/packages/core/src/schema.rs
+++ b/packages/core/src/schema.rs
@@ -82,13 +82,13 @@ table! {
     }
 }
 
 table! {
     property (id) {
         accountId -> Nullable<Uuid>,
-        address -> Jsonb,
+        // address -> Jsonb,
         buildPeriod -> Nullable<Text>,
         buildingLegalStatus -> Nullable<Text>,
         commonSpaces -> Nullable<Text>,
         energyClass -> Nullable<Text>,
         equipments -> Nullable<Text>,
         gasEmission -> Nullable<Text>,
@@ -161,13 +161,13 @@ table! {
 table! {
     user (id) {
         authId -> Nullable<Text>,
         email -> Text,
         firstName -> Nullable<Text>,
         lastName -> Nullable<Text>,
-        address -> Nullable<Jsonb>,
+        // address -> Nullable<Jsonb>,
         photoURL -> Nullable<Text>,
         role -> Nullable<Text>,
         id -> Uuid,
         phoneNumber -> Nullable<Text>,
         accountId -> Nullable<Uuid>,
     }
